code/modules/paperwork/faxmachine.dm

```diff
@@ -0,0 +1,58 @@
+
+//The IA Agent's Fax Machine
+
+/obj/machinery/faxmachine
+	name = "fax machine"
+	desc = "Used to send messages to your corporate overlords (and promptly get fired for writing fanfiction about their mothers)."
+	circuit = /obj/item/circuitboard/machine/faxmachine
+	icon = 'icons/obj/library.dmi'
+	icon_state = "fax"
+	density = TRUE
+	idle_power_usage = 30
+	active_power_usage = 200
+	COOLDOWN_DECLARE(static/important_action_cooldown)
+
+/obj/machinery/faxmachine/attack_hand(mob/living/user)
+	. = ..()
+	add_fingerprint(user)
+	if(!COOLDOWN_FINISHED(src, important_action_cooldown))
+		to_chat(user, "<span class='warning'>The subspace communications transmissions system is on cooldown!</span>")
+		return
+	if(panel_open)
+		return
+
+	var/associates = (obj_flags & EMAGGED) ? "the Syndicate" : "Central Command"
+	var/msg = capped_multiline_input(user, "Enter a message to be faxed to [associates]", "Fax machine", )
+
+	if(!msg)
+		return
+
+	if(!(obj_flags & EMAGGED))
+		message_centcom(msg, user)
+		to_chat(user, "<span class='notice'>Message transmitted to Central Command.</span>")
+	else
+		message_syndicate(msg, user)
+		to_chat(user, "<span class='danger'>SYSERR @l(19833)of(transmit.dm): !@$ MESSAGE TRANSMITTED TO SYNDICATE COMMAND.</span>")
+
+	override_cooldown()
```

Comment:
What does this do?

Reply:
i put the proc in the wrong place somehow but its supposed to reset the messaging cooldown when CC responds back, fixed now